### PR TITLE
Working swagger support

### PR DESCRIPTION
**Pull request by PeytonBoggs**
This branch supports a working Swagger UI. The local CRUD functions all have annotations that allow Swagger to execute them using custom parameters.

Screenshot 2023-06-28 at 2 56 43 PM

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,4 +41,11 @@ SQL:
         use tasks;
 
     Get table:
-        SELECT * FROM tasks;
\ No newline at end of file
+        SELECT * FROM tasks;
+
+Swagger:
```
Comment (matt-j-deasy) [inline README.md:49]: Love to see some README updates.

**Review by matt-j-deasy — APPROVED** [review]
Looks good.

**Review by matt-j-deasy — CHANGES_REQUESTED** [review]
WAIT A SECOND. WHeres the PR description? This could use a brief description and maybe a screenshot of the swagger docs working

**Comment by PeytonBoggs** [comment]
This branch supports a working Swagger UI. The local CRUD functions all have annotations that allow Swagger to execute them using custom parameters.

Screenshot 2023-06-28 at 2 56 43 PM

**Review by matt-j-deasy — APPROVED** [review]

**Pull request merged**